traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -252,10 +260,10 @@
             "title": "Global Max TPS",
             "desc": "The maximum transactions this delivery service can serve across all EDGE caches before traffic will be diverted to the bypass destination. For a DNS delivery service, the Bypass Ipv4 or Ipv6 will be used (depending on whether this was a A or AAAA request), and for HTTP delivery services the Bypass FQDN will be used."
           },
-	  "fqPacingRate": {
```

Comment:
can you look at why this formatting seems different?

Reply:
I accidentally left a tab in there -- fixed it.